wagtail/admin/views/generic/chooser.py

```diff
@@ -296,7 +297,7 @@ class CreationFormMixin(ModelLookupMixin, PreserveURLParametersMixin):
     def get_permission_policy(self):
         if self.permission_policy:
             return self.permission_policy
-        elif self.model_class:
+        elif issubclass(self.model_class, Model):
```

Comment:
Looking at the definition of `model_class` in `ModelLookupMixin`, it seems possible that `model` is `None`. Calling `issubclass(None, Model)` will cause a `TypeError: issubclass() arg 1 must be a class`. We should probably do something like

```suggestion
        elif self.model_class and issubclass(self.model_class, Model):
```

Reply:
Good catch! Applied in d51da2b85bb4e0a00de1a1810dd4989df2853977.